axon-framework/deadlines/event-schedulers.md

```diff
@@ -41,9 +43,19 @@ As such, it is a *hard requirement* to use Axon Server as your Event Store solut
 Just as the `QuartzEventScheduler`, the `AxonServerEventScheduler` is a reliable and enterprise-worthy implementation of the `EventScheduler` interface.
 Creating a `AxonServerEventScheduler` can be done through its builder, whose sole requirement is the `AxonServerConnectionManager`.
 
-It is important to note that the `JubRunrEventScheduler`, `QuartzEventScheduler` and `AxonServerEventScheduler` all should use the [event `Serializer`](../serialization.md#event-serialization) to serialize and deserialize the scheduled event.
+The `DbScheduler` is a reliable and enterprise-worthy implementation.
```

Comment:
I think it might be nicer to start this sentence differently from the JobRunr description.
Perhaps something like this:

```suggestion
The `DbSchedulerEventScheduler` is similar in convenience compared to the JobRunr- and the Axon Server `EventScheduler` implementations.
```